docker-compose-production.yml

```diff
@@ -0,0 +1,58 @@
+
+version: '3'
+services:
+  postgres:
+    image: postgres:11
+    ports:
+      - "5432:${DB_PORT}"
+    environment:
+      POSTGRES_USER: ${DB_USERNAME}
+      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_DB: ${DB_DATABASE}
+    networks:
+      - infra_postgres
+    volumes:
+      - pg-data:/var/lib/postgresql/data
+
+  server:
+    build:
+      dockerfile: "./Dockerfile"
+      context: "server/"
+    ports:
+      - "3000:${SERVER_PORT}"
+    environment:
+      SERVER_PORT: ${SERVER_PORT}
+      DB_TYPE: ${DB_TYPE}
+      DB_HOST: "postgres"
+      DB_PORT: ${DB_PORT}
+      DB_USERNAME: ${DB_USERNAME}
+      DB_PASSWORD: ${DB_PASSWORD}
+      DB_DATABASE: ${DB_DATABASE}
+    networks:
+      - infra_postgres
+      - infra_api
+    depends_on: ["postgres"]
+
+  client:
+    build:
+      dockerfile: "./Dockerfile"
+      context: "client/"
+    ports:
+      - "9000:${CLIENT_PORT}"
+      - "8080:8080"
+    environment:
```

Comment:
where are these variables used? Nginx doesn't seem to be reading them and there is nothing else running in this container that I can see